components/Cards/SchoolCard/__tests__/BackSchoolCard.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import BackSchoolCard from '../BackSchoolCard';
+
+const locations = [
+  {
```

Comment:
since the BackSchoolCard wouldn't normally be renderable with just one location, let's add a second location to the test data. I suggest doing multiple locations in each of the tests. Consider doing 
```
[ { ...locations[0] } ]
```
if you want to test a component with a list of 1 (potentially valuable in the front school card).